src/api/distinctionApi.tsx

```diff
@@ -16,13 +17,26 @@ const getUserDistinctions = async () => {
   return (await api.get(`Distinction/UserDistinctions`)).data;
 };
 
-const getAllUsersDistinctions = async (searchedData: string, page: number, pageSize: number) => {
+/*const getAllUsersDistinctions = async (searchedData: string, page: number, pageSize: number) => {
```

Comment:
It is best practise to remove commented code. Cause it can cause a lot of confusion later. 
So, please, remove it

Reply:
Done